pkg/commands/compute/deploy_test.go

```diff
@@ -160,18 +165,20 @@ func TestDeploy(t *testing.T) {
 			name: "service version is active, clone version error",
 			args: args("compute deploy --service-id 123 --token 123 --version 1"),
 			api: mock.API{
-				CloneVersionFn: testutil.CloneVersionError,
-				ListVersionsFn: testutil.ListVersions,
+				CloneVersionFn:      testutil.CloneVersionError,
+				GetServiceDetailsFn: getServiceDetailsWasm,
```

Comment:
There were quite a few existing tests that needed to be fixed so they mocked `GetServiceDetailsFn`, which was a new API call I added for validating the user's service type.